test/e2e/specs/various/navigation.spec.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Navigation block', () => {
```

Comment:
I think this should be called something like 'Classic menus' or just 'Menus', as it doesn't involve the navigation block.